app/assets/javascripts/discourse/app/widgets/header-contents.js

```diff
@@ -4,6 +4,13 @@ import hbs from "discourse/widgets/hbs-compiler";
 createWidget("header-contents", {
   tagName: "div.contents.clearfix",
   template: hbs`
+    {{#unless this.site.mobileView}}
```

Comment:
TGX added a new `site.desktopView` property, you could use that and do `(and prop1 prop2 prop3)`?